lib/Tool/Admin.php

```diff
@@ -202,6 +222,9 @@ public static function isMaintenanceModeScheduledForLogin(): bool
         return false;
     }
 
+    /**
+     * @deprecated
+     */
     public static function scheduleMaintenanceModeOnLogin(): void
```

Comment:
The replacement for `scheduleMaintenanceModeOnLogin` might be missing. 
See also https://github.com/pimcore/admin-ui-classic-bundle/pull/274#pullrequestreview-1633791218

Reply:
there's no replacement as it will be removed in Pimcore 12

Reply:
then suggesting to add the removal note after the `@deprecated` tag